lib/vtls/schannel.c

```diff
@@ -1631,6 +1631,7 @@ schannel_connect_step3(struct Curl_cfilter *cf, struct Curl_easy *data)
       args.data = data;
       args.idx = 0;
       args.certs_count = certs_count;
+      args.result = CURLE_OK;
```

Comment:
I don't understand this code, so I'm not sure about this one.

At a lower level, the same default result is set. The issue is that it's one level _too_ low, and a call above it misses it an leaves it uninitialized. So I set it here at the most upper level as a fix. But, defaulting to `OK` in a security setting feels a bit uncomfortable.